dev/com.ibm.ws.microprofile.config.1.3_fat/publish/files/refreshVariables/server.xml

```diff
@@ -0,0 +1,33 @@
+<server description="Server for testing variable Config">
```

Comment:
is this server.xml the same as the previous one?

Reply:
I've tweaked it to make clear the distinction between testing a changed server.xml "variable" attribute and appProperties "property" element.